embedchain/factory.py

```diff
@@ -50,7 +50,7 @@ def create(cls, provider_name, config_data):
 
 class EmbedderFactory:
     provider_to_class = {
-        "azure_openai": "embedchain.embedder.openai.OpenAIEmbedder",
+        "azure_openai": "embedchain.embedder.azureopenai.AzureOpenAIEmbedder",
```

Comment:
Name will also be changed here as well -> `"embedchain.embedder.azure_openai.AzureOpenAIEmbedder"`